app/src/main/java/net/squanchy/schedule/view/OtherEventItemView.kt

```diff
@@ -0,0 +1,67 @@
+package net.squanchy.schedule.view
+
+import android.content.Context
+import android.support.annotation.DrawableRes
+import android.util.AttributeSet
+import android.widget.ImageView
+import android.widget.TextView
+import net.squanchy.R
+import net.squanchy.schedule.domain.view.Event
+import org.joda.time.format.DateTimeFormat
+
+class OtherEventItemView @JvmOverloads constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int = R.attr.cardViewDefaultStyle) :
```

Comment:
Split ctor params over multiple lines rather than having a line break before the `extends` clause

Reply:
8dc3afa1d7570bdd8de79b6d5fb7e391d8db53cc